src/solver/impls/arkode/arkode.cxx

```diff
@@ -221,17 +221,21 @@ int ArkodeSolver::init(int nout, BoutReal tstep) {
     if(expl){	//Use purely explicit solver
       output.write("\tUsing ARKode Explicit solver \n");
       {TRACE("Calling ARKodeInit");
-	if( ARKodeInit(arkode_mem, arkode_rhs, NULL, simtime, uvec) != ARK_SUCCESS  ) //arkode_rhs_e holds the explicit part, arkode_rhs_i holds the implicit part
-	  throw BoutException("ARKodeInit failed\n");
+        if (ARKodeInit(arkode_mem, arkode_rhs, nullptr, simtime, uvec) !=
```

Comment:
Not sure this formatting is an improvement; clang seems to get it a bit wrong sometimes